### PR TITLE
8.0 version changes

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: docker/install-deps [hunk 1]
```diff
@@ -106,11 +116,11 @@ if [ ! -f /usr/local/lib/libeatmydata.so ]; then
     rm -rf /tmp/libeatmydata
 fi
 
-# boost 1.59.0 needed for percona-server 5.7
+# needed for percona-server 5.7
 wget_loop 'boost_1_59_0.tar.gz' 'http://downloads.sourceforge.net/boost/boost/1.59.0/boost_1_59_0.tar.gz'
```
Comment (laurynas-biveinis) [inline docker/install-deps]: No need for 5.7 prerequisites?

File: docker/install-deps [hunk 2]
```diff
@@ -106,11 +116,11 @@ if [ ! -f /usr/local/lib/libeatmydata.so ]; then
...
 
-# boost 1.66.0 needed for percona-server 8.0
-wget_loop 'boost_1_66_0.tar.gz' 'http://downloads.sourceforge.net/boost/boost/1.66.0/boost_1_66_0.tar.gz'
+# needed for percona-server 8.0
+wget_loop 'boost_1_67_0.tar.gz' 'http://downloads.sourceforge.net/boost/boost/1.67.0/boost_1_67_0.tar.gz'
 
 # googletest 1.8.0 needed for percona-server 5.7
```
Comment (laurynas-biveinis) [inline docker/install-deps]: Redundant comment

File: jenkins/pipeline.groovy [hunk 3]
```diff
@@ -121,11 +114,13 @@ pipeline {
                 }
 
                 sh 'echo Prepare: \$(date -u "+%s")'
-                git branch: '5.7', url: 'https://github.com/Percona-Lab/ps-build'
+                git branch: 'fix-8.0-version', url: 'https://github.com/Percona-Lab/ps-build'
```
Comment (laurynas-biveinis) [inline jenkins/pipeline.groovy:117]: Will be changed to 8.0 on deployment?

File: jenkins/pipeline.groovy [hunk 4]
```diff
@@ -121,11 +114,13 @@ pipeline {
...
                 sh '''
-                    git reset --hard
-                    git clean -xdf
+                    sudo git reset --hard
```
Comment (laurynas-biveinis) [inline jenkins/pipeline.groovy:121]: Maybe a good idea to put the explanation for sudo here in a comment

File: local/build-binary [hunk 5]
```diff
@@ -133,6 +133,9 @@ if [[ "${WITH_TOKUDB}" = "ON" ]]; then
     if [[ "${ANALYZER_OPTS}" = *WITH_VALGRIND=ON* ]]; then
         CMAKE_OPTS+=" -DUSE_VALGRIND=ON"
     fi
+else
+    # temporary add workaround for PS-4742
```
Comment (laurynas-biveinis) [inline local/build-binary]: Could be no longer required, 8.0 now has both TokuDB and Tokubackup

File: local/checkout [hunk 6]
```diff
@@ -57,6 +57,5 @@ pushd $ROOT_DIR
 
     if [ "x${PURGE_TOKUDBBACKUP}" = "xtrue" ]; then
         rm -rf plugin/tokudb-backup-plugin/Percona-TokuBackup
-        mkdir plugin/tokudb-backup-plugin/Percona-TokuBackup
```
Comment (laurynas-biveinis) [inline local/checkout]: Why?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

File: jenkins/pipeline.groovy (hunk 3 above)
Comment (delgod) [reply jenkins/pipeline.groovy:117]: yes

File: jenkins/pipeline.groovy (hunk 4 above)
Comment (delgod) [reply jenkins/pipeline.groovy:121]: fixed

File: local/build-binary (hunk 5 above)
Comment (delgod) [reply local/build-binary]: fixed

File: local/checkout (hunk 6 above)
Comment (delgod) [reply local/checkout]: fixed

File: docker/install-deps (hunk 2 above)
Comment (delgod) [reply docker/install-deps]: comments have been removed

File: docker/install-deps (hunk 1 above)
Comment (delgod) [reply docker/install-deps]: comments have been removed

File: jenkins/pipeline.groovy (hunk 4 above)
Comment (delgod) [reply jenkins/pipeline.groovy:121]: fixed

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by delgod** [comment]
merged manually to `8.0` branch, without one commit https://github.com/Percona-Lab/ps-build/pull/35/commits/664a89a4a39d628477f293b59f2b49fa965d6bdf

**Pull request closed without merging**